app/models/task.rb

```diff
@@ -1,3 +1,19 @@
 class Task < ActiveRecord::Base
+  belongs_to :person
 
+  def change_complete_status(complete_request)
+    complete_request == "1" ? complete : uncomplete
+  end
+
+  def complete
+    update(completed_at: Time.now)
+  end
+
+  def uncomplete
+    update(completed_at: nil)
+  end
+
+  def owner
+    Person.find(self.person_id).name
+  end
```

Comment:
This looks like another name for `person`, which might become confusing in the future unless they have different meanings that I'm missing.